process/block/postprocess/basePostProcess.go

```diff
@@ -273,7 +276,8 @@ func (bpp *basePostProcessor) addIntermediateTxToResultsForBlock(
 	sndShardID uint32,
 	rcvShardID uint32,
 ) {
-	addScrShardInfo := &txShardInfo{receiverShardID: rcvShardID, senderShardID: sndShardID}
+	bpp.index++
```

Comment:
can we move this increment after L280 so we start counting from 0?
I can see that the index is used only internally for the comparing of the scrs, but we might want to use the index elsewhere in the future.